### ISSUE TITLE
BUG: Elterngeld eligibility

### ISSUE THREAD

**Issue by MImmesberger** [iza-institute-of-labor-economics/gettsim#697]
### Bug description

A quick look at `elternzeit_anspruch` makes me question whether eligibility for Elterngeld is correct. In particular, 

```python
m_elterngeld_mut_hh + m_elterngeld_vat_hh
            < elterngeld_params["max_monate_paar"]
```

sounds like both parents are able to take up Elterngeld in scenarios where only one parent can actually do so (e.g. suppose that `elterngeld_params["max_monate_paar"] = 14` and `m_elterngeld_mut_hh + m_elterngeld_vat_hh=13`).

**Comment by MImmesberger** [comment]
Also, `alter_monate_jüngstes_mitglied_hh <= elterngeld_params["max_monate_paar"]` is being used to check whether children aren't too old for the parents to receive Elterngeld. Would be better to target the children directly (via `p_id_elternteil`).

**Comment by hmgaudecker** [comment]
I don't think Elterngeld has ever been touched since the earliest days of GETTSIM. Or definitely does not live up to our quality standards.

Probably worth starting over at some point rather than patching here and there.

**Comment by MImmesberger** [comment]
Closed via #756
